blog/_posts/2023-01-24-scala-center-2023-roadmap.md

```diff
@@ -0,0 +1,247 @@
+command-line apps `sbt` and `scala` now behave consistently regardless of the installation process (`cs install` vs
+manual download of the release artifacts).
+
+### Community and Contributor Experience
+
+**We co-organized the [ScalaCon](http://www.scalacon.org/) online conference.** We were in charge of the program and
+the program committee.
+
+**We published videos about Scala 3.** [Let’s Talk About Scala 3](https://www.youtube.com/playlist?list=PLTx-VKTe8yLxYQfX_eGHCxaTuWvvG28Ml)
+is a series of videos where we share interesting, useful, and cool things related to Scala 3. We published 4 new
+videos in 2022.
+
+**We improved the code editing experience in [Scastie](https://scastie.scala-lang.org).** Scastie is the online Scala
+code editor that is used every day by one thousand developers to share code snippets in the community. We have 
+implemented “IDE features” such as autocompletion and showing type information under the mouse pointer.
+
+**We mentored new contributors to the Scala 3 compiler.** Diving into the compiler is not an easy task. The Compiler
+Academy organizes online pair-programming events to mentor new contributors on the compiler codebase. You can learn
+more about the Compiler Academy in [this blog article]({% link blog/_posts/2022-11-02-compiler-academy.md %}).
+
+**We implemented the building blocks of a new tool to detect incompatibilities between programs.** Guaranteeing the
+absence of incompatibilities between library dependencies is the cornerstone of a peaceful Scala ecosystem. We have
+implemented a prototype of a new tool to detect incompatibilities between two versions of a program (like
+[MiMa](https://github.com/lightbend/mima), but based on the TASTy representation of programs instead of JVM bytecode).
+Our work is based on [TASTy Query](https://github.com/scalacenter/tasty-query), which will be the basis of tools to
+perform static analysis of Scala programs (it is already used by the Metals debugger).
+
+## Roadmap for 2023 (and Beyond)
+
+It is now time to look forward and present our goals for 2023. Our mission remains the same: improve the experience of
+becoming and being a Scala developer, and help the community to build a rich ecosystem of libraries. Ultimately, our
+work will ensure the longevity of Scala by creating a reliable and resilient ecosystem.
+
+We have identified the priorities for 2023 through our discussions with the community (online or at conferences),
+with our [Advisory Board members](https://scala.epfl.ch/#advisory-board-member-list), and with the
+main organizations that are [behind Scala](https://www.scala-lang.org/community/#whos-behind-scala)
+([LAMP](https://lamp.epfl.ch), [Lightbend](https://lightbend.com), and [VirtusLab](https://virtuslab.com)). We are
+grateful to all of them.
+
+In the following subsections we remind our ongoing and recurring projects, and we present our most important goals as
```

Comment:
s/we remind our/we remind you of our/ , or if that seems wordy, replace "remind" with some other verb such as "highlight"

Reply:
Done in 6c333175